gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,3 +1,6 @@
+// SPDX-FileCopyrightText: 2020, Torsten Grote <t@grobox.de>
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Looks like comments here are made with `#`, but this file gets auto-generated and will be overridden with the next gradle upgrade.

Reply:
Yes, no need of comment headers on auto-generated files.